executor/mpp_gather.go

```diff
@@ -94,20 +95,24 @@ func (e *MPPGather) Open(ctx context.Context) (err error) {
 		_, e.kvRanges, err = plannercore.GenerateRootMPPTasks(e.ctx, e.startTS, e.mppQueryID, sender, e.is)
 		return err
 	}
-	coord := e.buildCoordinator()
+	planIDs := collectPlanIDS(e.originalPlan, nil)
+	coord := e.buildCoordinator(planIDs)
+	err = mppcoordmanager.InstanceMPPCoordinatorManager.Register(mppcoordmanager.CoordinatorUniqueID{MPPQueryID: e.mppQueryID, GatherID: uint64(e.id)}, coord)
```

Comment:
If we need support gather level cancel, should not use `e.id` as GatherID

Reply:
I see, I'll change it like mpp task id

Reply:
Done